tests/perf/main/install-many-snaps/task.yaml

```diff
@@ -62,7 +62,14 @@ execute: |
                 echo "already $NUM_SNAPS installed, now check other features"
                 break
             fi
-        done        
+        done
+    done
+
+    while true; do
+        if snap changes | NOMATCH " Doing.*Install \".*\" snap"; then
+            break
+        fi
+        sleep 1        
     done
```

Comment:
Same question about the retry limit and potentially using `retry` if we can

Reply:
well, I avoided the retry because in real devices (slow) it could take a long time installing and in the end of the while will be the kill-timeout of the test.